templates/registration/logout.html

```diff
@@ -0,0 +1,20 @@
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Logged out" %}{% endblock %}
+
+{% block content %}
+<div class="container">
```

Comment:
In  CSS the * represents all elements, BS containers can be applied 2 ways, container - has margins on both sides and anything placed within it maintains the same. Container-fluid spans the total width of the document as will anything placed inside it.